CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+# Contributing to Database
+
+Thank you for your interest in participating!
+
+There are many ways to contribute, beyond writing code or programming, by: logging bugs, reporting issues, and creating suggestions. To do so, please [create a ticket](https://github.com/shabados/database/issues/new/choose) in our issue tracker. See other ways to [Contribute](README.md#Contributing) or give [Feedback](README.md#Feedback).
+
+This document is for developers or programmers contributing to the source code of Database.
+
+**Table of Contents**
+
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Build](#build)
+  - [Run](#run)
+  - [Convert & Import/Export](#convert--importexport)
+  - [Docker](#docker)
+- [File/Folder Structure](#filefolder-structure)
+  - [Schema](#schema)
+  - [Data Folder](#data-folder)
+- [Workflow](#workflow)
+  - [Coding Guidelines](#coding-guidelines)
+  - [Scope](#scope)
+- [Thank you](#thank-you)
+
+## Getting Started
+
+If you wish to better understand how Database works or want to debug an issue: get the source, build it, and run it locally.
+
+### Prerequisites
+
+In order to download necessary tools, clone the repository, and install dependencies, you'll need network access. To view the database you'll need a database viewer (e.g. [DBeaver](https://dbeaver.io/)).
+
+You'll need the following:
+
+- [Git](https://git-scm.com/)
+- [Node.JS](https://nodejs.org) (If you need to manage multiple Node.JS versions, [use a node version manager](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) to install)
+
+Get the source code of `database` repo:
+
+```shell
+gh repo fork shabados/database --clone=true
+```
+
+**PROTIP**: Use the [`gh` cli tool from GitHub](https://cli.github.com/) to fork the repo to your GitHub account (if not already), clone it to your local machine, and set the appropriate remotes for origin and upstream with the above command.
+
+### Build
+
+Run `npm i` in the root directory. Build an sqlite file by entering `npm run build-sqlite`.
+
+### Run
+
+Usage:
+
+```shell
+npm run <command>
+```
+
+The commands are:
+
+```shell
+build-sqlite  # Generate an sqlite db in the build folder
+build-mysql   # Generate a mySQL db in the build folder

```

Comment:
Untrue - generates the data for mysql, given the correct environment variables. Does not become outputted as a file

Reply:
so what does it build exactly? perhaps you can suggest a better description, i'm not quite aware of this

Reply:
`Builds the source data and writes it to a running MySQL instance` perhaps?